nixos/modules/services/networking/anubis.nix

```
@@ -286,7 +286,7 @@ in
                 ]
               )
             then
-              "anubis"
+              "anubis/%N"
```

related to previous changes:
```suggestion
              "anubis/${instanceName name}"
```